addons/account/populate/account_move.py

```diff
@@ -116,6 +133,7 @@ def get_entry_line(label, balance=None):
                     'balance': balance,
                     'account_id': account.id,
                     'partner_id': partner_id,
+                    'tax_ids': get_taxes(random, values),
```

Comment:
Apparently, I can't comment out this line, because it gives lint error "get_taxes is not used"